tests/testthat/_snaps/verb-expand.md

```diff
@@ -4,7 +4,7 @@
       lazy_frame(x = 1, y = 1) %>% tidyr::expand(x, y)
     Output
       <SQL>
-      SELECT `x`, `y`
+      SELECT `LHS`.*, `RHS`.*
```

Comment:
If it's easy, maybe we should only use `*` if there is more than one variable selected?

Reply:
Yes, this looks a bit better now.

Reply:
Nice!